src/frontend/src/modules/MiniGrid/Dashboard.vue

```diff
@@ -584,6 +584,10 @@ export default {
   margin-right: 1rem;
   margin-top: 3rem;
 }
+.md-toolbar {
+  position: relative;
+  z-index: 10000 !important;
+}
```

Comment:
Hey @munyanezaarmel 

Is there a way without using `!important` ? It feels strange to me to set the `z-index` on the entire toolbar. Isn't there a way to just fix the datepicker?